oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java

```diff
@@ -80,6 +80,11 @@
      */
     @Getter
     private int dayStep = 1;
+    /**
+     * Since 8.2.0, the record day step is for record index rolling when the value of it is gt 0
```

Comment:
Java has a `@since` annotation, please use it.

Reply:
fix